indigo/indigo/src/main/scala/indigo/shared/scenegraph/BlankEntity.scala

```diff
@@ -77,6 +77,18 @@ final case class BlankEntity(
   def withRef(x: Int, y: Int): BlankEntity =
     withRef(Point(x, y))
 
+  def resizeTo(newSize: Size): BlankEntity =
+    this.copy(size = newSize)
+  def resizeTo(w: Int, h: Int): BlankEntity =
+    resizeTo(Size(width, height))
+  def withSize(newSize: Size): BlankEntity =
+    resizeTo(newSize)
+
+  def resizeBy(amount: Size): BlankEntity =
+    this.copy(size = size * amount)
```

Comment:
Ah, sorry, just noticed this. `resizeBy` is addition of a relative amount, so `*` should be `+`.

Reply:
Ok so only scale used *

Reply:
Correct, yes. The use-case for `resizeBy` is "However big it is, I need it 10x10 pixels bigger".